File: mobile/lib/features/auth/data/models/auth_response.dart
class AuthResponse {
  final String token;
  final String email;
  final String role;
  final int organisationId;

  AuthResponse({
    required this.token,
    required this.email,
    required this.role,
    required this.organisationId,
  });

  factory AuthResponse.fromJson(Map<String, dynamic> json) {
    return AuthResponse(
      token: json['token'],
      email: json['email'],
      role: json['role'],
      organisationId: json['organisationId'],
    );
  }
}

File: backend/platform-contracts/src/main/java/univ/lille/dto/auth/AuthResponse.java
package univ.lille.dto.auth;

import lombok.AllArgsConstructor;
import lombok.Data;
import univ.lille.enums.UserRole;

@Data
@AllArgsConstructor
public class AuthResponse {
    private  String token ;
    private String email ;
    private UserRole role ;
    private Long organisationId ;

}
